lib/mbta.js

```diff
@@ -2,10 +2,64 @@
 
 // Code here.
 
+// stopsBetweenStations('Red Line', 'South Station', 'Red Line', 'Alewife')
 const stopsBetweenStations = (startLine, startStation, endLine, endStation) => {
+  let mbtaStops = {
+      'Red': [
+        'South Station',
+        'Park Street',
+        'Kendall',
+        'Central',
+        'Harvard',
+        'Porter',
+        'Davis',
+        'Alewife'
+      ],
+      'Green': [
+        'Government Center',
+        'Park Street',
+        'Boylston',
+        'Arlington',
+        'Copley',
+        'Hynes',
+        'Kenmore'
+      ],
+      'Orange': [
+        'North Station',
+        'Haymarket',
+        'Park Street',
+        'State',
+        'Downtown Crossing',
+        'Chinatown',
+        'Back Bay',
+        'Forest Hills'
+      ]
+  };
+
+  // scenario where we are on the same line simple subtraction
+  // scenario where we are on different lines
+  // // we have park street connection to account for with addition of stops for end Station
+  // if lines same this else that
+
+  if (startLine === endLine) {
+    let start = mbtaStops[startLine].indexOf(startStation);
+    let stop = mbtaStops[endLine].indexOf(endStation);
+    return Math.abs(stop - start)
+  } else {
+    let start = mbtaStops[startLine].indexOf(startStation);
+    let stop = mbtaStops[endLine].indexOf(endStation);
+
```

Comment:
There is no need to re-declare start and stop in each part of the if statement. Next time just declare them outside of the statement and it will still have access to the variables in each part of the conditional.